catroid/src/main/res/values-ko/strings.xml

```diff
@@ -1672,7 +1672,7 @@ https://share.catrob.at/ website 여기를 통해 계정과 정보를 삭제할
     <string name="formula_editor_logic_comparison">비교 연산</string>
     <string name="formula_editor_object_general">일반 속성</string>
     <string name="formula_editor_object_movement">운동 속성</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations